File: pruebareactts/src/models/Eleccion.ts

export type EleccionTipo = 'nacional' | 'departamental' | 'municipal';


export interface Eleccion {
    /** Identificador único de la elección (primary key del backend) */
    id: string;
    /** Tipo de elección */
    tipo: EleccionTipo;
    /** Fecha de la elección en formato ISO (YYYY-MM-DD) */
    fecha: string;
    /** Descripción de la elección */
    descripcion: string;
}

File: admElectoral/admElec/models/eleccion.py
from django.db import models

class Eleccion(models.Model):
    TIPO_CHOICES = [
        ('nacional', 'Nacional'),
        ('departamental', 'Departamental'),
        ('municipal', 'Municipal'),
    ]
    tipo = models.CharField(max_length=20, choices=TIPO_CHOICES)
    fecha = models.DateField()
    descripcion = models.TextField()

    def __str__(self):
        return f"{self.tipo} - {self.fecha}"
